js/src/modals/ExecuteContract/DetailsStep/detailsStep.js

```diff
@@ -141,6 +142,24 @@ export default class DetailsStep extends Component {
           );
           break;
 
+        case 'bool':
+          const boolitems = ['false', 'true'].map((bool) => {
+            return (
+              <MenuItem
+                key={ bool }
+                value={ bool }
+                label={ bool }>{ bool }</MenuItem>
+            );
+          });
+          inputbox = (
+            <Select
```

Comment:
Simple - we don't have a proper wrapped component that fits in with the l&f of the system which includes the labels and hints (not applicable here) as it does for the other inputs. As soon as we bring in a toggle, it raises all kinds of other issues where the specific field won't fit with the rest of the generated UI at this point. So for 1.4 it will be the dropdown.